screenshot-test/App.screenshot.js

```diff
@@ -0,0 +1,755 @@
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: geneQuery
+              }
+            );
+            break;
+          case `${SERVER_URL}api/private/search/variants/biological?hugoSymbol=ABL1`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: biologicalVariants
+              }
+            );
+            break;
+          case `${SERVER_URL}api/private/utils/portalAlterationSampleCount`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: sampleCount
+              }
+            );
+            break;
+          case `${SERVER_URL}api/v1/evidences/lookup?hugoSymbol=ABL1&evidenceTypes=GENE_SUMMARY`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: evidenceSummary
+              }
+            );
+            break;
+          case `${SERVER_URL}api/v1/evidences/lookup?hugoSymbol=ABL1&evidenceTypes=GENE_BACKGROUND`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: evidenceBackground
+              }
+            );
+            break;
+          case `${SERVER_URL}api/private/search/variants/clinical?hugoSymbol=ABL1`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: clinicalVariants
+              }
+            );
+            break;
+          case `https://www.genomenexus.org//ensembl/transcript`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: genomenexusTranscript
+              }
+            )
+            break;
+          case `https://www.genomenexus.org//ensembl/canonical-transcript/hgnc/?isoformOverrideSource=uniprot`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: genomenexusCanonicalTranscript
+              }
+            )
+            break;
+          case `${SERVER_URL}api/private/utils/variantAnnotation?hugoSymbol=ABL1&referenceGenome=GRCh37&alteration=BCR-ABL1%20Fusion`:
+            request.respond(
+              {
+                status: 200,
+                contentType: 'application/json',
+                body: variantAnnotation
+              }
+            );
+            break;
+          default:
+            request.continue();
+        }
+      }     
+    });
+    await page.goto(`${CLIENT_URL}`);
+    await page.evaluate(() => {
+      localStorage.setItem('localdev', 'true');
```

Comment:
Can we move this method to an util? I can see it be used frequently in the future.

Reply:
Actually it seems cannot be moved to an util, or the `localStorage` will ne undefined.